spell/spell-reviewer-goerli-checklist.md

```diff
@@ -49,6 +49,7 @@ Spell Actions:
   * [ ] Constructor args ok (e.g. `vat`, `dai`, `dog`, ...)
     * [ ] Match [ChainLog](https://chainlog.makerdao.com/)
   * [ ] Wards ok (pause proxy relied, deployer denied)
+    * [ ] Check if the contract requires to rely the ESM in the spell (to allow to deny pause proxy during Emergency Shutdown, via `denyProxy`).
```

Comment:
I don't like the wording here

I would say


```
Check whether the contract requires auth on the ESM (in order to allow de-authing the pause proxy during Emergency Shutdown, via `denyProxy`).
```

Or something worded more like this (if the above isn't correct)

Reply:
18cf91c